netweaver/setup/keepalive.sls

```diff
@@ -0,0 +1,12 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+# Based on https://launchpad.support.sap.com/#/notes/1410736
+# And https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse#2d6008b0-685d-426c-b59e-6cd281fd45d7
+
+{% for sysctl_key, sysctl_value in netweaver.sysctl_values.items()  %}
+
+{{ sysctl_key }}:
+  sysctl.present:
```

Comment:
As per the SAP note, we need to pass the `-w` flag when setting the new values. But I could see the salt fn supporting other parameters